tests/acceptance/features/apiSpaces/changeSpaces.feature

```diff
@@ -577,3 +577,23 @@ Feature: Change data of space
     Then the HTTP status code should be "404"
     When user "Carol" sends PATCH request to the space "Project Jupiter" of user "Alice" with data "{}"
     Then the HTTP status code should be "404"
+
+  @env-config
+  Scenario: space member with role 'Space Editor Without Versions' edits the space
+    Given the administrator has enabled the permissions role "Space Editor Without Versions"
+    And these users have been created with default attributes and without skeleton files:
+      | username |
+      | Carol    |
+    And user "Alice" has sent the following space share invitation:
+      | space           | Project Jupiter               |
+      | sharee          | Carol                         |
+      | shareType       | user                          |
+      | permissionsRole | Space Editor Without Versions |
```

Comment:
do we have a similar test for  | permissionsRole | Space Editor |. How about testing both on same scenario using outline

Reply:
step `the administrator has enabled the permissions role "<role>"` should allow other roles as well.